core/src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/MarthaDosResolverSpec.scala

```diff
@@ -3,13 +3,12 @@ package org.broadinstitute.dsde.rawls.dataaccess
 import akka.actor.ActorSystem
 import akka.stream.ActorMaterializer
 import akka.testkit.TestKit
-import org.broadinstitute.dsde.workbench.service.Sam.convertScalaFuture
 import org.scalatest.concurrent.{Eventually, ScalaFutures}
 import org.scalatest.time.{Seconds, Span}
 import org.scalatest.{FlatSpecLike, Matchers}
 import scala.concurrent.ExecutionContext.Implicits.global
 
-class MarthaDosResolverSpec extends TestKit(ActorSystem("MarthaDosResolverSpec")) with FlatSpecLike with Eventually with Matchers {
+class MarthaDosResolverSpec extends TestKit(ActorSystem("MarthaDosResolverSpec")) with FlatSpecLike with Eventually with ScalaFutures with Matchers {
```

Comment:
I unignored the test to see what would happen.  I did not get the Jackson error - which i could imagine might be caused by the dependency on `Sam.convertScalaFuture` / service-tests

Getting past that step, I got a different error:
```
org.broadinstitute.dsde.rawls.RawlsExceptionWithErrorReport: ErrorReport(rawls,http error calling uri https://us-central1-broad-dsde-staging.cloudfunctions.net/martha_v1,Some(502 Bad Gateway),List(),List(),None)
        at org.broadinstitute.dsde.rawls.util.HttpClientUtils.$anonfun$executeRequestUnmarshalResponse$2(HttpClientUtils.scala:42)
```